python/lib/dcm2bids_imaging_pipeline_lib/nifti_insertion_pipeline.py

```diff
@@ -635,7 +635,7 @@ def _register_violations_log(self, violations_list, file_rel_path):
             'TarchiveID': self.dicom_archive.id,
             'MincFile': file_rel_path,
             'PatientName': self.json_file_dict['PatientName'],
-            'CandID': self.session.candidate.cand_id,
+            'CandidateID': self.session.candidate.id,
```

Comment:
@maximemulder this was a bug highlighted by the automated tests in place so I fixed it here.